homeassistant/components/broadlink/sensor.py

```diff
@@ -76,11 +78,16 @@ def device_info(self):
             "sw_version": self._device.fw_version,
         }
 
+    @callback
+    def update_data(self):
+        """Update data."""
+        if self._coordinator.last_update_success:
```

Comment:
This will keep the sensors running smoothly even if we miss an update or two. The value will only be unknown after three minutes without a response from the device.